lib/board.rb

```diff
@@ -0,0 +1,19 @@
+# Understands how to represent a board for tic-tac-toe
+
+class Board
+  attr_reader :grid
+
+  def initialize
+    @grid = Array.new(3) { Array.new(3) { :" " } }
```

Comment:
Reliability: To avoid letting the state get into an erroneous state, try to consistently use the same methods to set the same state.  eg This line could use `fill_at` to set the state for each position if `fill_at` were modified.